spec/service_providers/webhooks_create_spec.rb

```diff
@@ -252,4 +252,49 @@
       expect(subject.success).to be true
     end
   end
+
+  context "when a valid webhook with a JSON body is submitted with a uuid" do
+    before do
+      params.merge!(uuid: '9999')
+      pact_broker
+        .given("the 'Pricing Service' and 'Condor' already exist in the pact-broker")
+        .upon_receiving("a request to create a webhook with a JSON body for a consumer and provider and a uuid")
+        .with(
+            method: :put,
+            path: '/webhooks/9999/provider/Pricing%20Service/consumer/Condor',
+            headers: put_request_headers,
+            body: request_body)
+        .will_respond_with(success_response)
+    end
+
+    it "returns a CommandResult with success = true" do
+      expect(subject).to be_a PactBroker::Client::CommandResult
+      expect(subject.success).to be true
+      expect(subject.message).to eq "Webhook \"A title\" created"
+    end
+  end
+
+  context "when only a uuid is specified" do
+    before do
+      params.merge!(uuid: '9999')
+      params.delete(:consumer)
+      params.delete(:provider)
+      mock_pact_broker_index(self)
+
+      pact_broker
+        .upon_receiving("a request to create a webhook with a JSON body and a uuid")
+        .with(
+            method: :put,
+            path: placeholder_path('pb:webhooks/9999'),
```

Comment:
Have I put the uuid (ie `9999`) in the correct place here? I'm assuming that the value `placeholder_path('pb:webhooks')` (which in the test returns `/HAL-REL-PLACEHOLDER-PB-WEBHOOKS`) would in production actually be `/webhooks` (which we would get by looking up the HAL relation from the index)

Reply:
The placeholder thing is an extra level of complication that you don't need right now. The URL should just be `/webhooks/9999`.

Reply:
Done :)